app/config/config_test.yml

```diff
@@ -7,6 +7,7 @@ parameters:
     api_url_public: http://api.elifesciences.org
     api_mock: '%kernel.cache_dir%/guzzle/fixtures'
     crm_url: http://crm.elifesciences.org/crm/civicrm/
+    feature_can_always_use_hypothesis: '%feature_can_always_use_hypothesis.override%'
```

Comment:
This is a bit annoying, but can't think of another way to enable/disable from the tests. (Using environment variables for parameters would have helped.)